package.json

```diff
@@ -28,19 +28,19 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "hexo-util": "^0.6.2",
-    "marked": "^0.6.1",
-    "strip-indent": "^2.0.0"
+    "hexo-util": "^0.6.3",
+    "marked": "^0.6.2",
+    "strip-indent": "^3.0.0"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
-    "eslint": "^5.9.0",
+    "chai": "^4.2.0",
+    "eslint": "^5.16.0",
     "babel-eslint": "^10.0.1",
     "eslint-config-hexo": "^3.0.0",
     "istanbul": "^0.4.5",
-    "mocha": "^6.0.0"
+    "mocha": "^6.1.4"
   },
   "engines": {
-    "node": ">=6.9.0"
+    "node": ">=8"
```

Comment:
`>=8.6.0` is better. It's the same with hexojs/hexo#3598

Reply:
ping @tomap

Reply:
done